$OpenBSD: patch-src_libicalvcal_vcc_h,v 1.1 2011/10/02 08:55:40 ajacoutot Exp $

Prevent warning when including this header:
    vcc.h:69:5: warning: "INCLUDEMFC" is not defined

--- src/libicalvcal/vcc.h.orig	Sat Oct  1 12:30:50 2011
+++ src/libicalvcal/vcc.h	Sat Oct  1 12:30:58 2011
@@ -66,7 +66,7 @@ will get a link error.
 */
 
 
-#if INCLUDEMFC
+#ifdef INCLUDEMFC
 extern VObject* Parse_MIME_FromFile(CFile *file);
 #else
 extern VObject* Parse_MIME_FromFile(FILE *file);
